### ISSUE TITLE
Add ability to parse folders and categorize games based on folders

### ISSUE THREAD

**Issue by hackmoore** [Hubro/clipface#17]
As the title says, would be awesome if I could do something like:

/clips/halo/clip1.mp4
/clips/halo/clip2.mp4
/clips/apex/clip1.mp4
/clips/apex/clip3.mp4

and the table will show a game name if the folders are present.

**Comment by Hubro** [comment]
I will definitely add folder support at some point. I haven't thought of a nice way to present folders in the user interface yet though. I don't want to put directories into the clip list together with the clips, that seems messy as hell.

I'm open to suggestions.

**Comment by hackmoore** [comment]
I was thinking something similar to the above structure, supporting only one level of directories and then displaying the directory in a separate column, but only if folders are used

**Comment by Hubro** [comment]
Oh, you're suggesting just adding the directory name in a new column? No hierarchical browsing of any kind?

**Comment by hackmoore** [comment]
Yeah that was the idea, seems simpler than implementing a directory browsing

**Comment by bigmak40** [comment]
Folder or directory support would be a huge value add. I'm currently running two separate containers to split up files and would love to just be able to use folders inside of one.

**Comment by Hubro** [comment]
I agree, this feature is on the short list. However, I'm extremely busy with other projects at the moment, so I don't have any capacity to work on Clipface. If anybody wants to contribute a pull request, I'll take the time to review it :slightly_smiling_face: Otherwise I very likely won't have time to work on Clipface until a vacation comes up.
